Makefile

```diff
@@ -54,7 +54,7 @@ verify-readme:
 
 .PHONY: validate-krew-manifest
 validate-krew-manifest: $(VALIDATE_KREW_MAIFEST)
-	$(VALIDATE_KREW_MAIFEST) -manifest dist/stern.yaml -skip-install
+	$(VALIDATE_KREW_MAIFEST) -manifest dist/krew/stern.yaml -skip-install
```

Comment:
:memo: The manifest path was changed in https://github.com/goreleaser/goreleaser/commit/76ce66c060341cd55983e68624c17f4e46377619.